Makefile

```diff
@@ -83,18 +83,18 @@ pre_commit_nodeps: ## Run pre-commit tests without installing dependencies
 	pushd $(BASEDIR); \
 	git config --global safe.directory '*'; \
 	if [ "x$(USE_VENV)" ==  'xyes' ]; then \
-		${HOME}/test-python/bin/pre-commit run --all-files 2>&1 | ansi2txt | tee $(LOG_DIR)/pre_commit.log; \
+		${HOME}/test-python/bin/pre-commit run --all-files 2>&1 | ansi2txt | tee $(LOG_DIR)/$@.log; \
```

Comment:
iirc those files were saved in zuul output. I think it's related to the pod run, it might be good to ensure they are still around...

Reply:
'k, so the artifacts link is broken indeed:
https://softwarefactory-project.io/zuul/t/rdoproject.org/build/5dc22b012af340058d46be298cecbc29
click on "Artifacts", then "[pre_commit logfile](https://logserver.rdoproject.org/78/2278/ab80f77ebfbd745f4e61b7a67198774dc433b1c6/github-check/cifmw-pod-pre-commit/5dc22b0/pre_commit.log)"

This comes from:
https://github.com/openstack-k8s-operators/ci-framework/blob/5fa99bbb78d4cab5246d6c711bf9c08e19043a1d/ci/playbooks/pod-jobs.yml#L44-L53